frontend/packages/dev-console/src/components/import/git/GitSection.scss

```diff
@@ -0,0 +1,3 @@
+.odc-expand-collapse > div:first-child > button:first-child {
```

Comment:
This will be removed but FYI the naming does not conform with our practices as the name should match the name of the component. eg `odc-git-section__expand-collapse`